jobs/build/incremental/Jenkinsfile

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env groovy
+
+OCP_VERSIONS = [
+        "4.0",
+        "3.11",
+        "3.10",
+        "3.9",
+        "3.8",
+        "3.7",
+        "3.6",
+        "3.5",
+        "3.4",
+        "3.3",
+        "3.2",
+        "3.1",
+]
+
+
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+    if (env.WORKSPACE == null) {
+        env.WORKSPACE = pwd()
+    }
+}
+
+def clean_comma_list(str) {
+    if (str == "") {
+        return ""
+    }
+    str = str.replaceAll(',', ' ')
+    return str.split().join(',')
+}
+
+def safeArchiveArtifacts(List patterns) {
+    for (pattern in patterns) {
+        try {
+            archiveArtifacts allowEmptyArchive: true, artifacts: pattern
+        } catch (err) {
+            println "Failed to archive artifacts like ${pattern}: ${err}"
+        }
+    }
+}
+
+def mail_failure(Map params) {
+    def err = params.err ? params.err.getMessage() : "(unknown)"
+    def subject = params.get("subject", "Problem with incremental build for ${MINOR_VERSION}")
+    def stage = params.stage ? "in stage '${params.stage}' " : ""
+    def extra_body = params.get("extra_body", "")
+    try {
+        mail(
+                to: "${MAIL_LIST_FAILURE}",
+                from: "aos-cicd@redhat.com",
+                replyTo: 'aos-team-art@redhat.com',
+                subject: subject,
+                body: """\
+Encoutered an error ${stage}while running ${env.JOB_NAME}:
+${err}
+
+Jenkins job: ${env.BUILD_URL}
+Console output: ${env.BUILD_URL}console
```

Comment:
Why extra 'console' at end?

Reply:
It is a link to the console output of the job. The job URL ends in `/`